CalendarLite.js

```diff
@@ -430,7 +430,7 @@ define([
 			//		protected
 			evt.stopPropagation();
 			evt.preventDefault();
-			for(var node = evt.target; node && !node.dijitDateValue; node = node.parentNode){
+			for(var node = evt.target; node && !node.dijitDateValue && node.dijitDateValue !== 0; node = node.parentNode){
```

Comment:
Could `!node.dijitDateValue && node.dijitDateValue !== 0` be simplified to `node.dijitDateValue !== 0`?